source/includes/fundamentals/code-snippets/indexes/atlasVectorSearch.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/v2/bson"
+	"go.mongodb.org/mongo-driver/v2/mongo"
+	"go.mongodb.org/mongo-driver/v2/mongo/options"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Retrieves your Atlas connection string
+	uri := os.Getenv("MONGODB_ATLAS_URI")
```

Comment:
I'm noticing that encoding/json and os are not in the imports list. That seems kind of strange, is it just how documentation code is written?

Reply:
Adding these imports.